<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>goods_history</class>
 <widget class="QDialog" name="goods_history">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoodsHistory</string>
  </property>
  <widget class="QTableWidget" name="table_g">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>991</width>
     <height>561</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_l">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>921</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ISum : 0  OSum : 0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fy_c">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="fm_c">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="fd_c">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tm_c">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="td_c">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ty_c">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="search_p">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
